release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2224,3 +2224,43 @@ If results return `Permission denied`, iptables or your kernal might need upgrad
+
+link:https://access.redhat.com/solutions/5945951[{product-title} 4.7.6 container image list]
+
+[id="ocp-4-7-6-bug-fixes"]
+==== Bug fixes
+
+* Previously, an error occurred when loading the topology page. Consequently, the page would not load. This update guards against the error so that the topology page loads successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940437[*BZ#1940437*])
+
+* When upgrading from 4.6 to 4.7, the hostname set by the vsphere-hostname service was only applied on installation of the node. If the hostname was not statically set prior to upgrading, the hostname might have been lost. This update removes the condition which allowed the vsphere-hostname service to only run when a node is installed. As a result, vsphere-hostnames are no longer lost when upgrading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1943143[*BZ#1943143*])
+
+* Previously, link:https://bugzilla.redhat.com/show_bug.cgi?id=1936587[*BZ#1936587*] set the global CoreDNS cache max TTL to 900 seconds. Consequently, NXDOMAIN records received from upstream resolvers were cached for 900 seconds. This update explicitly caches negative DNS response records for a maximum of 30 seconds. As a result, resolving NXDOMAINs are no longer cached for 900 seconds. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1943826[*BZ#1943826*])
+
+* Previously, the `growpart` script did not consider in-place LUKS rootfs reprovisioning as `requiring growing`. Consequently, machines which enabled in-place LUKs encryption ended up with rootfs too small. With this update, the `growpart` script (now `ignition-ostree-growfs`) considers in-place LUKS rootfs reprovisioning as  `requiring growing`. As a result, machines which enable in-place LUKs encryption have rootfs that take up all available space. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1941760[*BZ#1941760*])
+
+* Previously, the `prjquota` kernel argument was dropped if rootfs reprovisioning (e.g. LUKS) was enabled. Consequently, disk space quota management features in {product-name} would break. With this update, the `prjquota` kernel argument is now retained even if rootfs is reprovisioned. As a result, {product-name} features dependent on that rootfs mount option now work. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940966[*BZ#1940966*])
+
+[id="ocp-4-7-6-features"]
+==== Features
+
+[id="ocp-4-7-6-baremetal-operator-enhancement"]
+===== BareMetal Operator Enhancement
+This update adds new capabilities to the BareMetal Operator that allows for different reboot modes to be utilized. This provides a path for clients to quickly power down systems for remediation purposes and to recover workloads as quickly as possible in the event of a node failure. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1936407[*BZ#1936407*].
```

Comment:
- s/allows/allow
- s/utilized/used
- Style guide recommends comma separation of longer clauses, as in the second sentence here.